src/Pages/HomePage.js

```diff
@@ -0,0 +1,40 @@
+import { Box, Text, VStack, Image, useColorModeValue, Stack } from "@chakra-ui/react";
+
+import { NavBar } from "../components";
+
+export const HomePage = () => {
+  const theme = useColorModeValue("dark", "light");
+
+  return (
+    <Box textAlign="center" fontSize="xl">
+      <VStack>
+        <NavBar />
+        <VStack spacing={8}>
+          <Stack
+            spacing={8}
+            mt={20}
+            pl="10"
+            pr="10"
+            direction={{ base: "column", lg: "row"}}
```

Comment:
As a fan of Chakra UI I understand what these props are but in the case of other developers do you mind using their full names?